### PR TITLE
Ft/fix service

### PR DESCRIPTION
**Pull request by christelle-gihozo**
made the yellow sticky image sticky on the whole body( should continue to be sticky even when you reach the footer)
lg:
lg-yellow

md:
md1-yellow

sm:
md-yellow

remove the overflow
before:
overflow-lg

after:
no-overflow

### CODE REVIEW

File: src/sections/service.html [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-  <img class="lg:w-56 md:w-48 w-28" src="../../img/vector-6.png" alt="yellow-vector" />
-</div>
+
 <section
-  class="bg-primaryColor px-8 pb-16 lg:grid lg:min-h-[48.75rem] lg:min-w-[71.973rem] lg:grid-cols-2 lg:grid-rows-1 lg:place-content-center lg:justify-end lg:gap-48 lg:pb-16 lg:pr-6 lg:pt-36 md:ml-32"
+  class="bg-primaryColor grid grid-rows-1 gap-20 py-32 lg:grid-cols-2 lg:gap-60 md:gap-20 relative px-10 lg:px-36 md:px-20"
 >
   <div
-    class="w-full space-y-16 font-outfit text-2xl font-medium md:text-4xl lg:col-span-1 lg:row-span-1 lg:text-5xl"
+    class="flex flex-col gap-6 text-2xl font-outfit font-medium lg:text-4xl md:text-4xl lg:gap-12"
```
Comment (Kevin-gram) [inline src/sections/service.html]: Checkout the class arrangements

File: src/sections/service.html [hunk 2]
```diff
@@ -1,11 +1,9 @@
-<div class="flex items-center justify-center pb-16 sticky top-0 pr-10 lg:pl-28 z-40">
-  <img class="lg:w-56 md:w-48 w-28" src="../../img/vector-6.png" alt="yellow-vector" />
-</div>
+
 <section
-  class="bg-primaryColor px-8 pb-16 lg:grid lg:min-h-[48.75rem] lg:min-w-[71.973rem] lg:grid-cols-2 lg:grid-rows-1 lg:place-content-center lg:justify-end lg:gap-48 lg:pb-16 lg:pr-6 lg:pt-36 md:ml-32"
+  class="bg-primaryColor grid grid-rows-1 gap-20 py-32 lg:grid-cols-2 lg:gap-60 md:gap-20 relative px-10 lg:px-36 md:px-20"
```
Comment (Kevin-gram) [inline src/sections/service.html]: Checkout the class arrangement

File: src/sections/service.html [hunk 3]
```diff
@@ -20,25 +18,16 @@
...
-        best possible way. From analysis to implementation, we accompany you
-        every step of the way, ensuring that every measure is effective.
-      </p>
-    </div>
-  </div>
-</section>
+  <div class="relative md:flex md:justify-end lg:mx-5 w-[100%]">
+    <img src="/img/Vector 5.png" alt="" class="w-fit h-[70vh] md:w-[45vh] md:h-[80vh] lg:w-[100%] lg:h-[100%]" />
+    <p
+      class="absolute px-10 text-lg leading-5 text-primaryColor font-outfit font-light top-24 lg:pl-16 lg:top-32 md:top-52 md:pl-64 md:pr-14 lg:pr-8 md:text-2xl lg:text-lg 2xl:text-2xl md:leading-7 lg:leading-5"
```
Comment (Kevin-gram) [inline src/sections/service.html]: Arrange classes well

**Review by Kevin-gram — CHANGES_REQUESTED** [review]
I left some comments about  class arangements  feel free to use prettier and wrap it up !..

**Comment by christelle-gihozo** [comment]
I have arranged the classes

**Review by Kevin-gram — APPROVED** [review]
looks good now

**Pull request merged**